controllers/ManagementController.php

```diff
@@ -283,5 +302,37 @@ private function updateSocialMedia(ModifySocialMediaInformationsForm $formModify
             throw new CannotSaveException($socialMedia);
         }
     }
+
+    /**
+     * @param User $user
+     * @return bool
+     * @throws \PHPMailer\PHPMailer\Exception
+     */
+    private function mailToUserPasswordCreation(User $user)
+    {
+        $mail = Util::getConfiguredMailerForMailhog();
+        $mail->addAddress($user->email);
+        $mail->isHTML(false);
+        $mail->CharSet = 'UTF-8';
+
+        $mail->Subject = "Account's creation for " . $user->firstname . " " . $user->lastname;
+        $mail->Body = 'Click on the following link to create your password : http://127.0.0.1:8000/site/validate-account/' . $user->token ;
+
+        try {
+            if(!$mail->send()) {
+                return false;
+            }
+        } catch(Exception $e) {
+            return false;
```

Comment:
It is a bad practice to return false when an exception is caught, because you lose the error information. Your mailToUserPasswordCreation method should simply throw the exception, and you should optionally catch it in the main function which is calling this method.